asset_dashboard/static/js/components/maps/SelectAssetsMap.js

```diff
@@ -79,14 +81,11 @@ function SelectAssetsMap(props) {
         body: JSON.stringify(data)
     }).then((response) => {
       if (response.status == 201) {
-        // TODO: this reloads the page but clears the user search...
-        // Need to come up with way to reload by rehydrating the previous state
+        setAjaxMessage('Assets successfully saved.')
         location.reload()
-        // TODO: show success message
-        // https://stackoverflow.com/questions/13256817/django-how-to-show-messages-under-ajax-function
       }
     }).catch(error => {
-      // TODO show error message
+      renderMessage('An error occurred saving the selected assets. Please try again.', 'danger')
```

Comment:
The `renderMessage` function seems to do mostly the same thing as the AJAX message state approach. Is there a way to be consistent here, instead of doing the same thing two ways?